docs/create_dimensions.md

```diff
@@ -13,12 +13,35 @@ In order to successfully push your JSON schema into the Monitor platform, you wi
 1. In the hamburger menu on the left side of the screen of Maximo Asset Monitor, click `Connect`.
 2. To the right of where it says `Watson IoT Platform Service` click `View Details`.
 3. Now click the `Launch` button.
-4. 
-
-## Creating a Value Card
-
-## Uploading an Image
-
-## Creating an Image Card
-
-## Creating a Table Card
+4. Sign into the `vrvzh6` organization in the Watson IoT Platform Service window. You can locate the organization on the top right corner of your screen. Click in the red boxed area on your browser to change the organization if need be. ![Changing IoTP Org](img/cd2.png) 
+5. On the left side pop out menu, click on `Apps`.
+6. Next, click `+ Generate API Key` button.
+7. Click next until you hit the `Generate API Key` button. Nothing of importance for this exercise appears before this step.
+8. Make sure you copy and paste your API Key and Token in a safe place. You will not be able to retrieve the token again. You will need both the API Key and Token to push the dimensions to Monitor.
+## Pushing the Dimensions using Postman
+1. Figure out the hierarchy you will deploy. I used `Country`:US, `Site`:Cambridge, and `Asset`:Motor. You will be pushing key-value pairs associated with your devices into Monitor to create your dimensions.
```

Comment:
I try not to use the word "I". This is a document from IBM and while there is an I in IBM it shouldn't be a person.   For this I recommend including the Postman files as a collection so that they can just reuse them. Put them into a RESTAPIs folder perhaps?